numpy/add_newdocs.py

```diff
@@ -3315,7 +3315,7 @@ def luf(lamdaexpr, *args, **kwargs):
     >>> A = np.array([1, 256, 8755], dtype=np.int16)
     >>> map(hex, A)
     ['0x1', '0x100', '0x2233']
-    >>> A.byteswap(True)
+    >>> A.byteswap(inplace=True)
```

Comment:
While we're here, can we update the signature to include the default value of the argument?